### PR TITLE
Fix issues with protocol listener & refactor store

### PR DESCRIPTION
**Pull request by BelfordZ**

### CODE REVIEW

File: src/containers/MultiCreateTransaction/MultiCreateTransaction.js [hunk 1]
```diff
@@ -97,6 +97,27 @@ class MultiCreateTransaction extends React.Component {
     this.setTransaction('amount', amount);
   }
 
+  componentDidUpdate(prevProps) {
+    const {from, to, value, data } = prevProps;
+    const props = this.props;
+    if (from !== props.from || to !== props.to || value !== props.value || data !== props.data) {
```
Comment (BelfordZ) [inline src/containers/MultiCreateTransaction/MultiCreateTransaction.js]: should this check ALL the props?

File: src/store/store.js [hunk 2]
```diff
@@ -4,10 +4,11 @@ import createReduxLogger from 'redux-logger';
 import { createStore as createReduxStore, applyMiddleware, combineReducers } from 'redux';
 import { reducer as formReducer } from 'redux-form';
 import { ipcRenderer } from 'electron';
-import EthereumQRPlugin from 'ethereum-qr-code'
+import EthereumQRPlugin from 'ethereum-qr-code';
```
Comment (shanejonas) [inline src/store/store.js]: remove this

File: src/store/store.js [hunk 3]
```diff
@@ -29,8 +30,10 @@ import createLogger from '../utils/logger';
 import reduxLogger from '../utils/redux-logger';
 import reduxMiddleware from './middleware';
 
+import { onceServicesRestart, onceServicesStart } from './triggers';
+
 const log = createLogger('store');
-const qr = new EthereumQRPlugin()
+const qr = new EthereumQRPlugin();
```
Comment (shanejonas) [inline src/store/store.js]: not used

File: electron/protocol/index.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
+const { app, protocol, webContents } = require('electron');
+const log = require('../logger');
+
+const getMainWebContents = () => webContents
+      .getAllWebContents()
+      .find((webcontent) => !!webcontent.browserWindowOptions);
```
Comment (shanejonas) [inline electron/protocol/index.js]: formatting is messed up here

**Pull request merged**